akka-docs/src/main/paradox/scala/actors.md

```diff
@@ -480,50 +617,58 @@ actors which are traversed in the sense that if a concrete name lookup fails
 negative result is generated. Please note that this does not mean that delivery
 of that reply is guaranteed, it still is a normal message.
 
-@@snip [ActorDocSpec.scala]($code$/scala/docs/actor/ActorDocSpec.scala) { #identify }
+Scala
+:  @@snip [ActorDocSpec.scala]($code$/scala/docs/actor/ActorDocSpec.scala) { #identify }
+
+Java
+:  @@snip [ActorDocTest.java]($code$/java/jdocs/actor/ActorDocTest.java) { #import-identify #identify }
 
 You can also acquire an `ActorRef` for an `ActorSelection` with
 the `resolveOne` method of the `ActorSelection`. It returns a `Future`
-of the matching `ActorRef` if such an actor exists. It is completed with
+of the matching `ActorRef` if such an actor exists. @java[(see also 
+@ref:[Java 8 and Scala Compatibility](scala-compat.md) for Java compatibility).] It is completed with
 failure [[akka.actor.ActorNotFound]] if no such actor exists or the identification
```

Comment:
`scala-compat.md` is referenced here. Although it's enclosed in @java[], /scala/actors.md still tries to look up /scala/scala-compat.md. 

So I did made [`scala-compat.md` available both under /scala and /java](https://github.com/akka/akka/pull/23079/files#r119562860)